src/components/FlawContributors.vue

```diff
@@ -0,0 +1,166 @@
+<script setup lang="ts">
+import { onBeforeMount, ref } from 'vue';
+import { isDefined, watchDebounced } from '@vueuse/core';
+import sanitize from 'sanitize-html';
+import LabelDiv from './widgets/LabelDiv.vue';
+import useJiraContributors from '@/composables/useJiraContributors';
+import type { ZodJiraUserPickerType } from '@/types/zodJira';
+import { createCatchHandler } from '@/composables/service-helpers';
+
+const props = defineProps<{
+  taskKey: string,
+}>();
+
+const inputRef = ref<HTMLInputElement | null>(null);
+const query = ref<string>();
+const results = ref<ZodJiraUserPickerType[]>([]);
+
+const {
+  contributors,
+  isLoadingContributors,
+  searchContributors,
+  loadJiraContributors,
+  saveContributors,
+} = useJiraContributors(props.taskKey);
```

Comment:
Annotation for general discussion. Should non OSIDB stuff that is present on the FlawView be part of `FlawModel?

I had this exact question when worked on `internalComments`: initially made an individual composable, and at the end changed opinion and made them part of the flaw model with an unified `FlawCommentsModel` that handles both OSIDB and Jira comments ([PR](https://github.com/RedHatProductSecurity/osim/pull/285) currently on review)

My personal opinion is to make part of `FlawModel` every data we associate to a Flaw in the view, independently from the source. Reason is cause I think that way can help keeping things more organized in the frontend.

Reply:
Originally I expected OSIDB to supply all the data, but now that OSIM does some of its own data aggregation, a nice idea for the future would be to have separate OsidbFlawModel which just has OSIDB data and OsimFlawModel which contains all OSIM's aggregated data for a FlawModel